lib/Doctrine/ORM/Internal/Hydration/ObjectHydrator.php

```diff
@@ -44,6 +44,9 @@ class ObjectHydrator extends AbstractHydrator
     /** @var mixed[] */
     private $initializedCollections = [];
 
+    /** @var mixed[] */
```

Comment:
This is correct but I think you can be more precise.

```suggestion
    /** @var array<string, PersistentCollection> */
```

Reply:
I thought about this ofc but didn't want to differ from the same other collections. This file is to refactor anyway, aint' it?

Reply:
Yes, but I'm likely to be the one doing it. Please don't give me more work :pray:

Reply:
Huh, OK, no problem